agent/virustotal.py

```diff
@@ -30,7 +32,20 @@ def scan_file_from_message(message: msg.Message, api_key: str) -> Dict:
     return response
 
 
-def get_scans(response: Dict) -> Optional[Dict]:
+def scan_url_from_message(target: str, api_key: str) -> dict:
+    """Method responsible for scanning a file through the Virus Total public API.
+    Args:
+        target: url to scan.
+        api_key : Key for the virustotal api.
+    Returns:
+        response: The response of the Virus Total public API.
+    """
+    virustotal_client = virus_total_apis.PublicApi(api_key)
+    response = virustotal_client.get_url_report(target, timeout=TIMEOUT_REQUEST)
```

Comment:
I'm a bit concerned about the possibility of the API to crash due to the API rate limit